### PR TITLE
Added space before and after the slash(/) Behavioral / Emotional Need…

### PR DESCRIPTION
**Pull request by KShresthaCwds**
…s Domain name

## JIRA
https://osi-cwds.atlassian.net/browse/CANS-1044

## Description
Added space before and after the slash(/) Behavioral / Emotional Needs Domain.

## Tests
- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters and static analyzers for the project (SonarQube, eslint, rubocop, etc).
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.

### CODE REVIEW

**Comment by KShresthaCwds** [comment]
Retest this please

File: tools/instrument-parser/csv/cans_domains.csv [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
+4,Cultural Factors Domain,CUL,"These items identify linguistic or cultural issues for which service providers need to make accommodations (e.g., provide interpreter, finding therapist who speaks family's primary language, and/or ensure that a child/youth in placement has the opportunity to participate in cultural rituals associated with their cultural identity). Items in the Cultural Factors Domain describe difficulties that children and youth may experience or encounter as a result of their membership in any cultural group, and/or because of the relationship between members of that group and members of the dominant society. 
+It is it important to remember when using the CANS that the family should be defined from the individual youth's perspective (i.e., who the individual describes as part of her/his family). The cultural issues in this domain should be considered in relation to the impact they are having on the life of the individual when rating these items and creating a treatment or service plan.
```
Comment (wmitchell-cambria) [inline tools/instrument-parser/csv/cans_domains.csv]: I think this should all be on one line, right? Newlines usually indicate new entries in a csv.

Comment (KShresthaCwds) [reply tools/instrument-parser/csv/cans_domains.csv]: Good catch. Will change it again.

Comment (KShresthaCwds) [reply tools/instrument-parser/csv/cans_domains.csv]: As we have used e.g. and commas(,) in the Domain-Desc field, We have protected them by enclosing it in double quotes ("), which is considered as a single line.

**Pull request merged**